rasa/core/policies/rule_policy.py

```diff
@@ -23,8 +23,12 @@
     ACTION_BACK_NAME,
     ACTION_SESSION_START_NAME,
     RULE_SNIPPET_ACTION_NAME,
+    ACTION_DEFAULT_FALLBACK_NAME,
 )
 
+if TYPE_CHECKING:
+    from rasa.core.policies.ensemble import PolicyEnsemble
```

Comment:
I think this should solve the "Check Types" CI issue. (The file with `PolicyEnsemble` imports `RulePolicy`, so we have an import cycle. But as the one here is only for type checking, it should be fine to simple disable.)
```suggestion
    from rasa.core.policies.ensemble import PolicyEnsemble # pytype: disable=pyi-error
```